### PR TITLE
Add Avernic defender to emote clue info

### LINKED ISSUE #9832: "Blow a raspberry in the warrior's guild" is missing the Avernic defender
Not sure when it was changed, but it has been confirmed that the STASH unit has been updated to hold the Avernic Defender

image

### PR DESCRIPTION
**Pull request by Krysaczek**
Closes #9832 

There's a similar clue with dragon defender "Do a push up at the bank of the Warrior's guild. Beware of double agents! Equip a dragon battleaxe, a dragon defender and a slayer helm of any kind." - needs confirmation if Avernic works there too.

I couldn't find any info about trimmed defender, so I left it as is.

### CODE REVIEW

**Comment by Adam-** [comment]
Considering it says "a dragon defender of any kind" I think it would include the trimmed ones.

**Comment by Krysaczek** [comment]
> Considering it says "a dragon defender of any kind" I think it would include the trimmed ones.

That phrase belongs to a slayer helmet, I think. I found a comment in the code where standard dragon defender is considered as the only solution to Falo the Bard clue, that's another thing to confirm.

File: runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/EmoteClue.java [hunk 1]
```diff
@@ -138,7 +138,7 @@
...
+		new EmoteClue("Blow a raspberry at the bank of the Warrior's guild. Beware of double agents! Equip a dragon battleaxe, a slayer helm of any kind and a dragon defender or avernic defender.", "Warriors' guild", WARRIORS_GUILD_BANK_29047, new WorldPoint(2843, 3543, 0), RASPBERRY, item(DRAGON_BATTLEAXE), any("Defenders", item(DRAGON_DEFENDER), item(AVERNIC_DEFENDER)), any("Any slayer helmet", item(SLAYER_HELMET), item(BLACK_SLAYER_HELMET), item(GREEN_SLAYER_HELMET), item(PURPLE_SLAYER_HELMET), item(RED_SLAYER_HELMET), item(TURQUOISE_SLAYER_HELMET), item(SLAYER_HELMET_I), item(BLACK_SLAYER_HELMET_I), item(GREEN_SLAYER_HELMET_I), item(PURPLE_SLAYER_HELMET_I), item(RED_SLAYER_HELMET_I), item(TURQUOISE_SLAYER_HELMET_I), item(HYDRA_SLAYER_HELMET), item(HYDRA_SLAYER_HELMET_I))),
```
Comment (Hydrox6) [inline runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/EmoteClue.java]: `Defenders` is not descriptive at all

Comment (rbbi) [reply runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/EmoteClue.java]: Perhaps: "Blow a raspberry at the bank of the Warrior's guild. Beware of double agents! Equip a dragon battleaxe, a slayer helm of any kind _and a dragon or avernic defender._"

Comment (Hydrox6) [reply runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/EmoteClue.java]: That text is the clue itself (which needs to be exactly that), and not what I was talking about.

**Comment by Adam-** [comment]
Oh I had misread the clue. It does say `dragon defender or avernic defender`

**Pull request merged**